src/lib.rs

```diff
@@ -1,23 +1,18 @@
 use {
-  base64::{engine::general_purpose, Engine},
   bitcoin::{
-    absolute::LockTime,
-    blockdata::script,
-    consensus::{Decodable, Encodable},
-    key::{Keypair, TapTweak},
-    opcodes,
-    psbt::Psbt,
-    script::PushBytes,
-    secp256k1::{self, schnorr::Signature, Message, Secp256k1, XOnlyPublicKey},
-    sighash::{self, SighashCache, TapSighashType},
-    transaction::Version,
-    Address, Amount, Network, OutPoint, PrivateKey, PublicKey, ScriptBuf, Sequence, Transaction,
-    TxIn, TxOut, Witness,
+    absolute::LockTime, blockdata::script, opcodes, psbt::Psbt, script::PushBytes,
+    secp256k1::Secp256k1, transaction::Version, Address, Amount, Network, OutPoint, PrivateKey,
+    PublicKey, ScriptBuf, Sequence, Transaction, TxIn, TxOut, Witness,
   },
   bitcoin_hashes::{sha256, Hash},
-  std::{io::Cursor, str},
 };
 
+mod sign;
+pub use sign::{full_sign, simple_sign};
+
+mod verifier;
+pub use verifier::{full_verify, simple_verify};
```

Comment:
I prefer having the pub use combined and the modules underneath each other.
```suggestion
mod verifier;
pub use { 
  sign::{full_sign, simple_sign},
  verifier::{full_verify, simple_verify},
};
```